draft-ietf-aipref-vocab.md

```diff
@@ -46,19 +46,28 @@ informative:
 
 --- abstract
 
-This document proposes a standardized vocabulary of use cases that can be targeted when expressing machine-readable opt-outs related to Text and Data Mining (TDM) and AI training. The vocabulary is agnostic to specific opt-out mechanisms and enables declaring parties to communicate restrictions or permissions regarding the use of their digital assets in a structured and interoperable manner. It defines three key use cases—TDM, AI Training, and Generative AI Training—which can be referenced by opt-out systems to ensure consistent interpretation across different implementations.
+This document proposes a standardized vocabulary for expressing preferences related to how content is consumed by automated processing systems.
+This vocabulary allows for the creation of structured declarations about restrictions or permissions for use of digital assets.
```

Comment:
```suggestion
This vocabulary allows for the creation of declarations about restrictions or permissions for use of digital assets.
```

Reply:
structured is a feature of serialization, and this document doesn't cover serialization

Reply:
Yet.  My understanding of the defined scope of this document is that it will include serialization and processing.

Reply:
Yes and no @martinthomson .

My understanding was that we agreed to start the document with a general purpose vocabulary, and then we would use that in a "second part" which would be a specific serialization/processing use case (e.g., HTTP headers)

Reply:
The demarcation that I had in mind was that we'd describe a general abstract model here and an exemplary serialization.  The HTTP/robots.txt attachment document would normatively depend on that serialization, but other attachment methods might choose instead to normatively depend on the abstract model instead and define their own serialization.

Now that I spent more words, does that match your understanding?

Reply:
Yes, I agree with that...and means I continue to stand by comment about removing "structured".